lib/ansible/modules/network/haproxy.py

```diff
@@ -287,7 +308,7 @@ def get_state_for(self, pxname, svname):
         """
         data = self.execute('show stat', 200, False).lstrip('# ')
         r = csv.DictReader(data.splitlines())
-        state = tuple(map(lambda d: { 'status': d['status'], 'weight': d['weight'] }, filter(lambda d: (pxname is None or d['pxname'] == pxname) and d['svname'] == svname, r)))
+        state = tuple(map(lambda d: { 'status': d['status'], 'weight': d['weight']'scur': d['scur'] }, filter(lambda d: (pxname is None or d['pxname'] == pxname) and d['svname'] == svname, r)))
```

Comment:
Need to add comma
state = tuple(map(lambda d: { 'status': d['status'], 'weight': d['weight']', scur': d['scur'] }, filter(lambda d: (pxname is None or d['pxname'] == pxname) and d['svname'] == svname, r)))